e2e/testsuite/tx.go

```diff
@@ -222,16 +222,13 @@ func (s *E2ETestSuite) ExecuteAndPassGovV1Beta1Proposal(ctx context.Context, cha
 		panic("ExecuteAndPassGovV1Beta1Proposal must be passed a cosmos.CosmosChain")
 	}
 
-	proposalID := s.proposalIDs[chain.Config().ChainID]
-	defer func() {
```

Comment:
I still think this is used somewhere else, so unless we can get rid of the whole s.proposalIDs, I think we still need the defer below line 231.

If you can easily get the proposalID for the other use and get rid of it entirely, that is preferable :)

Reply:
Hello @gjermundgaraba , i updated, can you help me review it

Reply:
This looks correct to me! Thank you :)